container/options.go

```diff
@@ -216,6 +220,20 @@ func SplitPercent(p int) SplitOption {
 	})
 }
 
+// FixHeight sets the heights of the top and the bottom containers.
+// If an argument is equal to -1, then the height will be set automatically,
+// i.e. will fill the rest of the space that is available.
+func FixHeight(th int, bh int) SplitOption {
```

Comment:
It takes two integers as you can then stack containers without having unreachable cells.

Example:
```
c, err := container.New(
  t,
  container.SplitHorizontal(
    container.Top(
      container.Border(linestyle.Light),
      container.PlaceWidget(test),
    ),
    container.Bottom(
      container.SplitHorizontal(
        container.Top(
          container.Border(linestyle.Light),
          container.PlaceWidget(test),
        ),
        container.Bottom(
          container.SplitHorizontal(
            container.Top(
              container.Border(linestyle.Light),
              container.PlaceWidget(test),
            ),
            container.Bottom(
              container.Border(linestyle.Light),
              container.PlaceWidget(test),
            ),
            container.FixHeight(3, -1),
          ),
        ),
        container.FixHeight(3, -1),
      ),
    ),
    container.FixHeight(3, -1),
  ),
)
```
Code above will give us:

![Screenshot from 2019-05-05 23-28-43](https://user-images.githubusercontent.com/16340977/57200619-7f1a9b00-6f8e-11e9-9165-2ec7582be405.png)

where the first 3 widgets are fixed. While if `SplitFixed` only takes one argument, this would not be possible I think?

Reply:
Thank you for the example code and the screenshot, this helps a lot. I think we might be talking about the same thing. We aren't really using the second argument in your example, it is always set to -1.

If container.SplitFixed() takes just one argument (the number of cells for the top container on a horizontal split or the number of cells for the left container on a vertical split) the above code should work in exactly the same way? Unless of course I misunderstood something.

Reply:
Oh yeah, we're talking about the same thing.

Back to your previous comment on
> Are there any use cases that wouldn't be satisfying with such an option? It might be better if we focus on those and see if we really need two arguments.

Makes sense, having two arguments probably won't have much use case as most will want to fill the space. Will change it to `SplitFixed(cells int)` instead.